### PR TITLE
setup authentication for the random database.

### PR DESCRIPTION
**Pull request by Marianasilvago**
### What

Added authenticated user for topic api component test

### How to review

Check the code changes pass.

### Who can review

Anyone

### CODE REVIEW

File: features/steps/topic_component.go [hunk 1]
```diff
@@ -74,6 +81,42 @@ func NewTopicComponent(mongoFeature *componenttest.MongoFeature, zebedeeURL stri
...
+		IsSSL:                   false,
+		ConnectTimeoutInSeconds: 15,
+		QueryTimeoutInSeconds:   15,
+
+		Username:        "",
+		Password:        "",
+		ClusterEndpoint: getMongoURI,
+		Database:        databaseName,
+	}
+	mongoConnection, _ := dpMongoDriver.Open(mongoConnectionConfig)
```
Comment (daiLlew) [inline features/steps/topic_component.go:97]: Do you need to check this error? Presumably fail the test if this is not nil.

Comment (Marianasilvago) [reply features/steps/topic_component.go]: updated in the next commit.

**Review by daiLlew — APPROVED** [review]
